pkg/clicmd/root.go

```diff
@@ -130,7 +130,7 @@ func NewRoot(appFs afero.Fs, wd string, args []string) (*cobra.Command, error) {
 	rootCmd.PersistentFlags().CountP(flagVerbose, "v", "Increase verbosity. May be given multiple times.")
 	rootCmd.PersistentFlags().Set("logtostderr", "true")
 	rootCmd.PersistentFlags().Bool(flagTLSSkipVerify, false, "Skip verification of TLS server certificates")
-	rootCmd.PersistentFlags().String(flagDir, wd, "Ksonnet application root to use; Defaults to CWD")
+	rootCmd.PersistentFlags().String(flagDir, "", "Ksonnet application root to use; Defaults to CWD")
```

Comment:
i'm not sure you want this change in here?

Reply:
So I kinda hacked around this - make the docs with an empty string to avoid having my local paths appear in the docs then rebuilt with the original default